doc/release/0.4.rst

```diff
@@ -15,8 +15,6 @@ New Features
    :func:`armi.utils.hexagon.getIndexOfRotatedCell`
    (`PR#1846 <https://github.com/terrapower/armi/1846`)
 #. Allow merging a component with zero area into another component (`PR#1858 <https://github.com/terrapower/armi/pull/1858>`_)
-#. :meth:`armi.reactor.blocks.HexBlock.rotate` only rotates parameters that are marked with
```

Comment:
I can see the logic in removing this release note. Good catch!

Are you going to add your own release note?

Reply:
Yes

Reply:
Added